docs/README.md

```diff
@@ -11,7 +11,8 @@ to .NET applications without having to modify their source code.
 > The following documentation refers to the in-development version
 of OpenTelemetry .NET Automatic Instrumentation. Docs for the latest version
 ([1.0.0](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest))
-can be found [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v1.0.0/docs/README.md).
+can be found under [opentelemetry.io](https://github.com/open-telemetry/opentelemetry.io/tree/main/content/en/docs/instrumentation/net/automatic)
+or [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v1.0.0/docs/README.md).
```

Comment:
Do we want to maintain docs here if they're upstream now? Shouldn't we remove them?

Reply:
I think it is better if they stay here so that we update docs during development. Then, during releasing, it would be easy for us to update opentelemtry.io docs just by looking at diff.